app/script/view_model/panel/ConversationDetailsViewModel.js

```diff
@@ -194,14 +196,14 @@ z.viewModel.panel.ConversationDetailsViewModel = class ConversationDetailsViewMo
         : z.l10n.text(z.string.ephemeralUnitsNone);
     });
 
-    this.shouldUpdateScrollbar = ko
-      .computed(() => this.serviceParticipants() && this.userParticipants() && this.isVisible())
-      .extend({notify: 'always', rateLimit: {method: 'notifyWhenChangesStop', timeout: 0}});
-
     const addPeopleShortcut = z.ui.Shortcut.getShortcutTooltip(z.ui.ShortcutType.ADD_PEOPLE);
     this.addPeopleTooltip = ko.pureComputed(() => {
       return z.l10n.text(z.string.tooltipConversationDetailsAddPeople, addPeopleShortcut);
     });
+
+    this.shouldUpdateScrollbar = ko
+      .computed(() => this.serviceParticipants() && this.userParticipants() && this.isVisible())
```

Comment:
Since this doesn't have any side-effects, would be nice to convert it into a `pureComputed`